spacepy/irbempy/irbempy.py

```diff
@@ -1433,16 +1435,29 @@ def _get_Lstar(ticks, loci, alpha, extMag='T01STORM', options=[1,0,0,0,0], omniv
     nTAI = len(ticks)
     d = prep_irbem(ticks, loci, alpha, extMag, options, omnivals)
     nalpha = d['nalpha']
+    if d['kext'] != 5 or d['options'][4] != 0:
+        landi2lstar = False
+    no_shell_splitting = (nalpha == 0) or (nalpha == 1 and alpha[0] == 90)
         
-    if isinstance(alpha, numbers.Number): # no drift shell splitting
-        raise Exception
```

Comment:
This looks like this was a case SpacePy never actually handled before, is that accurate?

Reply:
I think it was some debug code to ensure the _get_Lstar function will not be called with only one value.